Include/Win/RestartManager.nsh

```diff
@@ -34,7 +34,7 @@ RestartManager.h
 !define RmCritical 1000 ; Application is critical system process where a reboot is required to restart
 
 !define RmForceShutdown           0x1 ; Force unresponsive applications and services to shut down after the timeout period
-!define RmShutdownOnlyRegistered 0x10 ; Only shudown apps if all apps registered for restart (RegisterApplicationRestart)

```

Comment:
This is actually from the Windows SDK

Reply:
Ah right. That would be a case where we need to fix upstream, but  the Win32 API documentation upstream appears to have been modified anyway:
[RmShutdown function (restartmanager.h)](https://docs.microsoft.com/en-us/windows/win32/api/restartmanager/nf-restartmanager-rmshutdown)
>  Shut down applications if and only if all the applications have been registered for restart using the [RegisterApplicationRestart](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-registerapplicationrestart) function

Reply:
I meant the typo is in the header file in the SDK, not on MSDN. I don't even think [they care](https://devblogs.microsoft.com/oldnewthing/20040705-00/?p=38573) nor do I know where to report it.